webknossos-datastore/app/com/scalableminds/webknossos/datastore/models/datasource/AdditionalAxis.scala

```diff
@@ -29,6 +29,15 @@ object AdditionalAxis {
       p => AdditionalAxis(p.name, Array(p.bounds.x, p.bounds.y), p.index)
     )
 
+  def fromProtoAsOpt(additionalAxisProtos: Seq[AdditionalAxisProto]): Option[Seq[AdditionalAxis]] = {
```

Comment:
```suggestion
  def fromProtoSeqAsOpt(additionalAxisProtos: Seq[AdditionalAxisProto]): Option[Seq[AdditionalAxis]] = {
```
(or something similar,) to make it clear that this is about multiple AdditionalAxis elements (since the caller only sees `AdditionalAxis.fromProtoAsOpt` which looks a lot like singular)